src/lib/StaticStore.ts

```diff
@@ -16,7 +17,7 @@ export class StaticStore {
     public request?: ParsedRpcRequest;
     public kind?: RequestType;
     public rpcState?: RpcState;
-    public keyguardRequest?: any;
+    public keyguardRequest?: KeyguardRequest;
```

Comment:
Can optionally also `import KeyguardClient from '@nimiq/keyguard-client` and then use `KeyguardClient.Request` as the type. I think this is how we do it in other places.

Reply:
Yes, but importing whole namespaces is actually not considered best practice.
I'd leave it like this.